ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-l3-port-channels.yml

```diff
@@ -176,9 +176,6 @@ tenants:
                   key: ocAzkT2JRNKrbPGT790Geg==
                 - id: 2
                   key: uVG6QXeIXN+QGY2DP4UCtA==
-                # Test when 'id' and 'key' are not set under message-digest keys.
-                # TODO: AVD 6.0.0 Make 'id' a primary key and 'key' required
```

Comment:
to the second reviewer - this comment was added before there was cleartext_key so the decision was taken to not make key required